### PR TITLE
Assignment 2: Convert to a plugin

### PR DESCRIPTION
**Pull request by rodrigo-medeiros**
Could've used the 'load one plugin' method to register, but since probably we're going to build more plugins, I found relevant to use the other one (passing an array).

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -15,15 +14,15 @@ internals.init = function () {
...
-        path: '/version',
-        config: {
-            description: 'Returns the version of the server',
-            handler: function (request, reply) {
+    server.register([
+        {
+            register: Version,
+            options: {}
+        }
+    ], function (err) {
```
Comment (chyld) [inline lib/index.js]: use Hoek

**Comment by hueniverse** [comment]
Don't try to predict the future. You made you code more complex now, when adding `[]` later will take literally seconds.

File: lib/index.js (hunk 1 above)
Comment (rodrigo-medeiros) [inline lib/index.js]: Good catch @chyld, fixed, thanks.

**Comment by rodrigo-medeiros** [comment]
I see @hueniverse, you're right. But, to be honest, I think that passing an array of objects to the `register` method makes the code much more _readable_, even though I know this is totally subjective.

**Pull request closed without merging**